src/js/internal/errors.ts

```diff
@@ -1,7 +1,34 @@
+const { SafeArrayIterator } = require("internal/primordials");
+
+const ArrayIsArray = Array.isArray;
+const ArrayPrototypePush = Array.prototype.push;
+
+class AbortError extends Error {
```

Comment:
this abort error will rarely be used.

can this be the $ABORT_ERR instead?

Reply:
will check uses